frontend/controller/actions/group.js

```diff
@@ -122,10 +122,12 @@ export default sbp('sbp/selectors/register', {
   },
   'gi.actions/group/updateSettings': async function (settings, groupId) {
     try {
+      this.dumbError.forced = 123 // DELETE THIS BEFORE MERGE
       const message = await sbp('gi.contracts/group/updateSettings/create', settings, groupId)
       await sbp('backend/publishLogEntry', message)
     } catch (e) {
-      throw new GIErrorUIRuntimeError(L('Failed to update group settings. {codeError}', { codeError: e.message }))
+      console.error('gi.actions/group/updateSettings failed!', e)
```

Comment:
Remember the `catch` in `GroupSettings.js` will also log this... so that's why we didn't include this `console.error` in the actions.

Reply:
Ignore for now, no big deal, we can go back through this later.